app/scripts/views/legal_copy.js

```diff
@@ -2,63 +2,36 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+/**
+ * A view to fetch and render legal copy. Sub-classes must provide
+ * a `copyUrl` where the copy template can be fetched, as well a
+ * `fetchError`, which is an error to display if there is a
+ * problem fetching the copy template.
+ */
+
 define(function (require, exports, module) {
   'use strict';
 
-  var BackMixin = require('views/mixins/back-mixin');
-  var BaseView = require('views/base');
-  var Cocktail = require('cocktail');
-  var xhr = require('lib/xhr');
-
-  // A view to fetch and render legal copy. Sub-classes must provide
-  // a `copyUrl` where the copy template can be fetched, as well a
-  // `fetchError`, which is an error to display if there is a
-  // problem fetching the copy template.
+  const BackMixin = require('views/mixins/back-mixin');
+  const BaseView = require('views/base');
+  const Cocktail = require('cocktail');
+  const xhr = require('lib/xhr');
 
-  var View = BaseView.extend({
-    initialize: function (options) {
+  const proto = BaseView.prototype;
+  const View = BaseView.extend({
+    initialize (options = {}) {
       this._xhr = options.xhr || xhr;
     },
 
-    afterRender: function () {
-      var self = this;
-      return self._xhr.ajax({
+    afterRender () {
+      return this._xhr.ajax({
         accepts: { text: 'text/partial' },
         dataType: 'text',
-        url: self.copyUrl
+        url: this.copyUrl
       })
-      .then(function (template) {
-        var $legalCopy = self.$('#legal-copy');
-
-        $legalCopy.html(template);
-
-        // data-visible-url is used to display the href in
-        // brackets next to the original text.
-        // Only show the href if the href is different from
-        // the text. See #2225.
-        //
-        // Links are only made visible from the main app. If
-        // the user browses directly to a TOS/PP page, the links
-        // are normal.
-        self.$el.addClass('show-visible-url');
-        self.$('a').each(function (index, element) {
-          element = $(element);
-
-          var href = element.attr('href');
-          var text = element.text();
-          // if the href is the same as the link text, don't convert.
-          if (href && href !== text) {
```

Comment:
The logic to only convert links if the href is different to the text was added to the external-links-mixin.